src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -15,125 +15,6686 @@
+                {
+                    _isWorgen = true;
+
+                    if (me->GetEntry() == NPC_PHIN_ODELIC)
+                        DoCastSelf(SPELL_BLOOD_ODELIC);
+                    else if (me->GetEntry() == NPC_BARTOLO_GINSETTI)
+                        DoCastSelf(SPELL_BLOOD_BARTOLO);
+                    else if (me->GetEntry() == NPC_LOREMASTER_DIBBS)
+                        DoCastSelf(SPELL_BLOOD_DIBBS);
+                    else if (me->GetEntry() == NPC_MAGISTRATE_HENRY_MALEB)
+                        DoCastSelf(SPELL_BLOOD_HENRY);
+                    else if (me->GetEntry() == NPC_CARETAKER_SMITHERS)
+                        DoCastSelf(SPELL_BLOOD_SMITHERS);
+                    else if (me->GetEntry() == NPC_SOPHIA_ZWOSKI)
+                    {
+                        DoCastSelf(SPELL_BLOOD_ZWOSKI);
+
+                        me->PlayDirectSound(SOUND_HOWLING);
+                    }
+
+                    _events.ScheduleEvent(EVENT_ACTOR_FENRIS_MORPH + 1, 1s);
+                    break;
+                }
+
+                case EVENT_ACTOR_FENRIS_MORPH + 1:
+                {
+                    me->HandleEmoteCommand(EMOTE_STATE_READY_UNARMED);
+
+                    if (me->GetEntry() == NPC_MAGISTRATE_HENRY_MALEB)
+                        _events.ScheduleEvent(EVENT_ACTOR_FENRIS_MORPH + 2, 2s);
+                    break;
+                }
+
+                case EVENT_ACTOR_FENRIS_MORPH + 2:
+                {
+                    if (Creature* agatha = me->FindNearestCreature(NPC_AGATHA_FENRIS, 60.0f))
+                        me->CastSpell(agatha, SPELL_GENERAL_TRIGGER_84107, true);
+                    break;
+                }
+
+                default:
+                    break;
+            }
+        }
+    }
+
+private:
+    EventMap _events;
+    ObjectGuid _playerGUID;
+    bool _isWorgen;
+};
+
+enum SylvanasWindrunnerCommandAndSepulcher
+{
+    QUEST_LORDAERON                         = 27098,
+    QUEST_TO_FORSAKEN_HIGH_COMMAND          = 27290,
+
+    SPELL_APPLY_INVIS_ZONE_1                = 83231,
+    SPELL_APPLY_INVIS_ZONE_4                = 84183,
+    SPELL_SUMMON_SYLVANAS_AND_HORSE         = 84128,
+    SPELL_SUMMON_FORSAKEN_WARHORSE          = 84164,
+    SPELL_LORDAERON_AURA                    = 84189, // NYI, after researching thoroughly, nothing seems to point what this SPELL_AURA_DUMMY should do. Maybe it is what forces the player to cast the rest of the spells.
+    SPELL_SUMMON_LORDAERON_ACTORS           = 84127,
+    SPELL_FLIGHT_OF_THE_VALKYR_FORWARD      = 84695,
+
+    SEAT_WARHORSE_SYLVANAS                  = 0
+};
+
+// Lady Sylvanas Windrunner - 44365
+struct npc_silverpine_sylvanas_fhc : public ScriptedAI
+{
+    npc_silverpine_sylvanas_fhc(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        if (me->GetPhaseShift().HasPhase(DEFAULT_PHASE) || me->GetPhaseShift().HasPhase(PHASE_FORSAKEN_HIGH_COMMAND_INTRODUCTION))
```

Comment:
This is no good approach

Reply:
Removing the phase thing because it is unnecessary in the end. The creature is only present in two places.